src/main/java/uk/gov/digital/ho/hocs/extracts/service/ExtractsEventService.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.digital.ho.hocs.extracts.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.digital.ho.hocs.extracts.core.exception.EntityCreationException;
+import uk.gov.digital.ho.hocs.extracts.repository.AuditRepository;
+import uk.gov.digital.ho.hocs.extracts.repository.entity.AuditEvent;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+import static net.logstash.logback.argument.StructuredArguments.value;
+import static uk.gov.digital.ho.hocs.extracts.core.LogEvent.EXTRACTS_EVENT_DELETED;
+import static uk.gov.digital.ho.hocs.extracts.core.LogEvent.EVENT;
+
+@Service
+@Slf4j
+public class ExtractsEventService {
+
+    private final AuditRepository auditRepository;
+    private static List<String> EXCLUDED_EVENT_TYPES = List.of(
+            "CASE_SUMMARY_VIEWED",
+            "CASE_VIEWED",
+            "SOMU_ITEMS_VIEWED",
+            "SOMU_ITEM_VIEWED",
+            "STANDARD_LINE_VIEWED",
+            "TEMPLATE_VIEWED");
+
+    @Autowired
+    public ExtractsEventService(AuditRepository auditRepository) {
+        this.auditRepository = auditRepository;
+    }
+
+    public AuditEvent createExtractsEvent(String correlationID, String raisingService, String auditPayload, String namespace, LocalDateTime auditTimestamp, String type, String userID) {
+        return createExtractsEvent(null, null, correlationID, raisingService, auditPayload, namespace, auditTimestamp, type, userID);
+    }
+
+    public AuditEvent createExtractsEvent(UUID caseUUID, UUID stageUUID, String correlationID, String raisingService, String auditPayload, String namespace, LocalDateTime auditTimestamp, String type, String userID) {
```

Comment:
This could return void but I don't want to get too far into refactoring just yet